AmazonChimeSDKDemo/AmazonChimeSDKDemo/CaptionCell.swift

```diff
@@ -23,8 +25,34 @@ class CaptionCell: UITableViewCell {
         self.contentView.backgroundColor = caption.speakerName.isEmpty ? .lightGray : .none
         speakerNameLabel.text = caption.speakerName
         speakerNameLabel.accessibilityIdentifier = caption.speakerName
-        captionContentLabel.text = caption.content
         captionContentLabel.accessibilityIdentifier = "caption-\(indexPath.row)"
+        // Highlight identified/redacted PIIs in green.
+        let  coloredString = NSMutableAttributedString.init(string: caption.content)
```

Comment:
two space on `let  coloredString` should be one

Reply:
will be updated.